src/styles/blocks/content.scss

```diff
@@ -0,0 +1,21 @@
+.content {
+  &__box {
+    margin: 0 auto;
+    padding: $page-padding 0;
+    max-width: $content-box-width;
+    min-width: $content-box-width;
+
+    display: flex;
+    flex-wrap: wrap;
+  }
```

Comment:
You could apply this styles to `content` class (except margins, you need to create BEM element for margins) and avoid additional div here. Also it's strange that we set outer distance between element and parent container